services/src/main/java/io/scalecube/services/ServiceScanner.java

```diff
@@ -31,7 +42,10 @@ private ServiceScanner() {
               List<ServiceMethodDefinition> actions =
                   Arrays.stream(serviceInterface.getMethods())
                       .filter(method -> method.isAnnotationPresent(ServiceMethod.class))
-                      .map(method -> new ServiceMethodDefinition(Reflect.methodName(method)))
+                      .map(
+                          method ->
+                              new ServiceMethodDefinition(Reflect.methodName(method))
+                                  .setTags(Reflect.serviceMethodTags(method)))
```

Comment:
Remove  method `setTags` (as well as `setAction`  method) and use another constructor `ServiceMethodDefinition(String action, Map<String, String> tags)`